pkgs/development/tools/language-servers/emmet-language-server/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, fetchpatch
+}:
+
+buildNpmPackage rec {
+  pname = "emmet-language-server";
+  version = "v2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "olrtg";
+    repo = pname;
+    rev = version;
```

```suggestion
    rev = "v${version}";
```
Once you get rid of `v` from version, you must add it here to match the tag